### DISCUSSION TITLE
quote highlighting changed in 2.5.1?

### DISCUSSION THREAD

**Discussion by mogplus88** [vkbo/novelWriter#2020] [Help] [answered]
Has the quote highlighting changed? It seems that double quoted and single quoted text is now highlighted the same way. It used to be different. I set up my own highlighting scheme ages ago, which looks like this.
```[Main]
name       = My Solarized Dark
author     = 
credit     = 
url        = 
license    = 
licenseurl = 

[Syntax]
background     =   7,  54,  66
text           = 253, 246, 227
link           =  38, 139, 210
headertext     = 147, 161, 161
headertag      =  42, 161, 152
emphasis       = 138, 139, 210
straightquotes =  211, 54, 130
doublequotes   = 142, 161,  52
singlequotes   =  42, 161, 152
hidden         = 147, 161, 161
keyword        = 133, 153,   0
value          = 203,  75,  22
spellcheckline = 203,  75,  22
errorline      = 220,  50,  47
replacetag     = 133, 153,   0
modifier       = 181, 137,   0
```
which used to highlight single quoted text in a mid blue, and double quoted text in yellow, but now they both appear in black.
Am I doing something wrong?
I use a Fedora based linux (Ultramarine) and the NW appimage, V2.5.1
Thanks, Ian

**Comment by vkbo** [comment, marked as answer]
If you make your own themes, you need to keep them up to date. Changes are recorded in the docs: https://novelwriter.io/docs/more_customise.html#custom-syntax-theme

In 2.5 the quotes highlighting changed into proper dialogue highlighting with new settings in Preferences. There are a few bugs with it, but I'm releasing new fixes this weekend.

**Reply by mogplus88** [reply]
Thanks for the quick reply. I've updated my theme, and with a bit of experimentation I've managed to figure out what everything means. 
Not sure if I've stumbled across a possible bug though. I set the alternative dialog symbols in preferences to "<" and ">", and that worked, the text with them wrapped around it was highlighted in the altdialog colour. Then I decided it might be easier to use a symbol that didn't involve having to use the Shift key (yes, I'm that lazy), so I changed them both to be backslash. That worked too, but oddly, the < and > symbols also still worked, but only if they are wrapped around one word. As soon as there is a space in the text, it doesn't highlight any more.
;-) Ian

**Reply by vkbo** [reply]
`<>` already has a meaning in novelWriter syntax, so they are highlighted. So when you set you alt dialog symbols to existing syntax, it's just a matter of which highlighting rule runs last.

It is not a well documented feature, but it is explained here: https://novelwriter.io/docs/project_overview.html#auto-replace-tab

**Reply by vkbo** [reply]
Also, when using backslash, keep in mind novelWriter does support a few escape sequences. They're documented a little bit here and there, but the complete set is `"\*", "\~", "\_", "\[", "\]", "\ "`.

**Comment by mogplus88** [comment]
Thanks for the tips, appreciate it. 
;-)
